packages/hint-doctype/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "hint to validate if the doctype is correct",
+  "devDependencies": {
+    "@hint/utils-tests-helpers": "^1.0.0",
+    "@types/debug": "0.0.30",
+    "@types/node": "8.0.14",
+    "ava": "^0.25.0",
+    "cpx": "^1.5.0",
+    "eslint": "^5.5.0",
+    "eslint-plugin-markdown": "^1.0.0-beta.7",
+    "eslint-plugin-typescript": "^0.12.0",
+    "markdownlint-cli": "^0.13.0",
+    "npm-link-check": "^2.0.0",
+    "npm-run-all": "^4.1.2",
+    "nyc": "^13.0.1",
+    "rimraf": "^2.6.2",
+    "hint": "^3.3.2",
```

Comment:
Update to the latest versions of all packages. E.g.: hint is 3.4.10, TypeScript 3.1.1, etc.

Reply:
I think that the newest version that I've seen in other packages is typescript@3.0.3 what is better? Update our packages to the latest version as others have in the project, or update to the latest version using some tools like `ncu` ?

Reply:
@greenkeeper[bot] should have updated those 🤔
I think it's best if you use the latest version in this package and we can take care of updating in a separate PR.

Reply:
Ok, I'm using as reference `hint-minified-js`